src/libm/sleefdp.c

```diff
@@ -27,6 +27,9 @@
 #define PI_C 1.2246467864107188502e-16
 #define PI_D 1.2736634327021899816e-24
 
+#define PI_A2 3.141592653589793116
```

Comment:
These coefficients are shared by both scalar and simd implementation. I recommend placing  them in a common (private) header file.